packages/StepTracker/StepTracker.modules.scss

```diff
@@ -0,0 +1,112 @@
+@import '~@tds/core-colours/colours';
+@import '~@tds/core-responsive/responsive';
+
+.container {
+  padding: 0;
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  margin: 1rem -1.5rem;
```

Comment:
Can we use `Box` instead of hard coded margins?

Reply:
I'll give it a shot.